docs/team/richiehx.md

```diff
@@ -5,42 +5,46 @@ title: "Richie's Project Portfolio Page"
 
 ### Project: TutorMate
 
-TutorMate helps private tuition teachers manage their students more easily. It allows faster access and storage of students’ contacts and data, providing better time management and productivity. The app will help to manage students for a single tuition teacher only. The app will only provide simple student analytics (without detailed student progress analysis).
+TutorMate helps private tuition teachers manage their students more easily. It allows faster access and storage of students’ contacts and data, providing better time management and productivity. The app will help to manage students for a single tuition teacher only. 
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to show the details of the selected student/lesson/task.
-  * What it does: allows the user to display the selected item in the show details panel at the right of the User Interface. It allows the user to show students while in the `STUDENT` list, show lessons while in the `SCHEDULE` list and show tasks while in the `TASK` list.
+* **New Feature**: Added the ability to show the details of the selected student/lesson/task. (Pull request [#53](https://github.com/AY2324S1-CS2103T-T11-3/tp/pull/53), [#138](https://github.com/AY2324S1-CS2103T-T11-3/tp/pull/138))
+  * What it does: allows the user to display the selected item in the show details panel at the right of the User Interface. It allows the user to show students while in the ___STUDENT list___, show lessons while in the ___SCHEDULE list___ and show tasks while in the ___TASK list___.
```

Comment:
Emm, how about change SCHEDULE, STUDENT and TASK to lower case to standardize? I thought Jung says we stick to either bold or italic, and do not do upper case.

Reply:
I approve first

Reply:
In my opinion, everything else just doesn't look good. You guys can try it out and I will standardise to you all.